commands/publish/index.js

```diff
@@ -146,8 +146,21 @@ class PublishCommand extends Command {
 
     const tasks = [
       () => this.getVersionsForUpdates(),
-      versions => {
-        this.updatesVersions = versions;
+      ({ versions, hasBreakingChange }) => {
+        let updatedVersions;
+        if (!this.project.isIndependent() && hasBreakingChange) {
```

Comment:
if one package has a breaking bump, all packages should bump together in a fixed version project